torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -2552,17 +2557,63 @@ def is_strict_for(v: VariableTracker):
         with tx.output.subtracer(fwd_fn, fwd_tracer), tx.strict_translation_mode(
             is_strict_for
         ):
-            (bwd_out, _), bwd_graph, bwd_freevars = speculate_subgraph(
-                tx,
-                bwd_fn,
-                bwd_args,
-                kwargs,
-                "autograd.Function",
-                enable_grad=False,
-                set_subgraph_inputs="manual",
-                restore_side_effects=False,
-                tracer=bwd_tracer,
-            )
+            try:
+                (bwd_out, _), bwd_graph, bwd_freevars = speculate_subgraph(
+                    tx,
+                    bwd_fn,
+                    bwd_args,
+                    kwargs,
+                    "autograd.Function",
+                    enable_grad=False,
+                    set_subgraph_inputs="manual",
+                    restore_side_effects=False,
+                    tracer=bwd_tracer,
+                )
+            except torch._dynamo.exc.Unsupported as e:
+                import traceback
+
+                if (
+                    len(e.real_stack) > 0
+                    and isinstance(e.real_stack[-1], traceback.FrameSummary)
+                    and e.real_stack[-1]._line is not None
+                    and ".is_contiguous()" in e.real_stack[-1]._line
+                    and e.real_stack[-1].name == "backward"
+                ):
```

Comment:
I’m unsure if this is the best way to specify the exception raised from an assert is_contiguous call. Alternatively, I could define a custom exception class, raise it when is_contiguous is called in the backward pass, and handle it here.